modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -93,6 +115,265 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanCreateEndpointWithSuccessAlert {
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
```

Comment:
Could you check all the steps for this test? Thanks! there are some steps missing.

Reply:
About creating an API endpoint on UI, we have two buttons 'Add API Endpoint' button and the 'Plus button', so it's a great way to test them, we need to click them here, don't remove them, please. you can use type here to input values

Reply:
hi @pedroamarall , the case we're testing is using `Add API Endpoint` button, and in APIBuilder.createAPIEndpoint it is using `PLUS` button to create a new endpoint, Please help update the createAPIEndpoint in APIBuilderUI.macro